code/core/system/proxy-fh16/src/CANMessageDataStore.cpp

```diff
@@ -35,9 +35,10 @@ namespace proxy {
 
 using namespace std;
 
-CanMessageDataStore::CanMessageDataStore(shared_ptr< automotive::odcantools::CANDevice > canDevice)
+CanMessageDataStore::CanMessageDataStore(shared_ptr< automotive::odcantools::CANDevice > canDevice, bool readOnly /* =false */)
```

Comment:
Here it says "false" still (bu in the comment). I suggest to remove this since it's an unnecessary point of redundancy. In fact I would even suggest to set de default value in the initialization list in the constructor to avoid logic in the header file.